### ISSUE TITLE
Size hints are honored

### ISSUE THREAD

**Issue by Chrysostomus** [PapyElGringo/material-shell#166]
Some apps like gnome-terminal request certain window sizes, and material-shell obliges. This leads to weird uneven gaps between windows. It would be better if size hints were not honored at all, or if it would be a toggle-able option. 

![Screenshot from 2020-03-07 23-10-32](https://user-images.githubusercontent.com/12002226/76152520-27c2eb00-60c9-11ea-8108-3854677a7b92.png)

**Comment by PapyElGringo** [comment]
I'm aware of this... Unfortunately Mutter seem to not allow us to do this. I don't know how to override this.

**Comment by Chrysostomus** [comment]
I think I have seen some other tiling extensions manage this. I'll see if I can find a working code example from some other project

**Comment by Chrysostomus** [comment]
I looked through other tiling extensions, tested manual resizing and looked to open an issue in gnome-terminal repo. I found that it is a hard wont fix there, and there is now working implementation under gnome. As you said, mutter simply does not permit. The relevant issue seems to be this: 

https://gitlab.gnome.org/GNOME/mutter/issues/869

So, the issue is probably not solvable within this project.
